src/components/notifications/NotificationsMenu.js

```diff
@@ -163,6 +174,26 @@ function NotificationsMenu(props) {
         },
     });
 
+    const [markAllSeenMutation] = useMutation(markAllSeen, {
+        onSuccess: () => {
+            setAllNotificationsSeen();
+        },
+        onError: (error) => {
+            showErrorAnnouncer(t("errorMarkAsSeen"), error);
```

Comment:
You've got the right idea that `showErrorAnnouncer` is a prop that gets passed to your component. The one thing you're missing is how your component gets that prop. Reviewing how HOCs work, the [code for our error HOC](https://github.com/cyverse-de/sonora/blob/master/src/components/utils/error/withErrorAnnouncer.js), and how other components are using that `withErrorAnnouncer` code should help.

Reply:
I think it should be correct now. I tested it by substituting the `onError` part with `onSuccess` and commenting out the original `onSuccess` and clicking Mark All As Seen in dev.

![image](https://user-images.githubusercontent.com/55575102/104758754-1a7c8100-571c-11eb-9a4a-37ab299e1132.png)

Reply:
Excellent!  

This is one way to test in dev sort of on the fly.  Another way you could test that might be faster and wouldn't require you to switch code back and forth is by using Storybook.  

If you recall, you're able to mock API calls in Storybook using [axios-mock-adapter](https://github.com/ctimmerm/axios-mock-adapter).  So far, we've mocked our API calls to always be successful by returning HTTP 200 and then providing a response with mock data.  

You can also test out your component by returning an unsuccessful response back.  There are examples in that mock axios repo and [here's](https://github.com/cyverse-de/sonora/blob/master/stories/notifications/Listing.stories.js) an example in our repo that Paul did for his portion of Notifications.  In this case, this story wanted an error to always show up the first time you try to mark a notification as seen, but then succeed after that, so when you run the story you can see both branches of pass/fail logic.

Reply:
Also, minor thing - since the text on the button says `Mark All as Read`, I would change the language here in the error message to match that: `Unable to mark notifications as read`